src/analytics.test.js

```diff
@@ -160,7 +160,6 @@ describe('analytics identifyAuthenticatedUser', () => {
     identifyAuthenticatedUser();
 
     expect(mockLoggingService.logError.mock.calls.length).toBe(1);
-    expect(mockLoggingService.logError).toBeCalledWith('UserId was not available for call to sendAuthenticatedIdentify.');
```

Comment:
FYI: I recently wrote a unit test that checked for the specific error and it failed because a different error was thrown first.  This would no longer catch that issue.  You could also grab the argument and check if it start with `Failed to sendAuthenticatedIdentify`.